dnnlib/comet_utils.py

```diff
@@ -0,0 +1,17 @@
+from comet_ml import Experiment
+import tensorflow as tf
+import json
+
+class CometLogger():
+    def __init__(self):
+            self.experiment = Experiment()
```

Comment:
You need to surround this with a try/except and give proper warning.

Reply:
Surrounded. I see it can also throw an error if API is missing.